frontend/test/metabase/scenarios/collections/collection-items-listing.cy.spec.js

```diff
@@ -121,18 +125,22 @@ describe("scenarios > collection items listing", () => {
       });
 
       toggleSortingFor(/Name/i);
+      cy.wait("@getCollectionItems");
+
       getAllCollectionItemNames().then(({ actualNames, sortedNames }) => {
         expect(actualNames, "sorted alphabetically reversed").to.deep.equal(
           sortedNames.reverse(),
         );
       });
 
       toggleSortingFor(/Name/i);
+      // Not sure why the same XHR doesn't happen after we click the "Name" sorting again?
```

Comment:
@kulyk any idea why this is the only occasion for which the `GET /collection/root/items` doesn't fire?

Reply:
I suppose the request doesn't fire because this one is cached. It's done the first time the page is open (retrieve collection content sorted by items name asc). Then the test clicks on "Name" to change the sorting to desc. Then when you click the name again, the sorting is changed back to asc. Since this request was already done, the FE doesn't request the data again but just uses a cached response